### ISSUE TITLE
How to force the Lyrics refreshing?

### ISSUE THREAD

**Issue by Jamy-L** [Felitendo/jellyfin-lyrics-plugin#3]
Hi, I really love the plugin, I have really been looking for this since a while !

Im encountering the following issue : I did a (successful) scan of my library, then realised that I would love to have .lrc files poping up next to my flacs. So I went to the library setting and checked the very last setting which says something like "Save lyrics next to audio files".

I made a rescan of the library but the .lrc were not generated. So I went to a specific song and manually deleted the lyric (Using the UI, there is an option to inspect lyrics, then delete them). I did so for all the songs of the album, and forced the rescan as advised in the Readme, but this time lyrics are not downloaded at all. I tried to delete and add again my whole library but it doesnt work either. Hence my question:
How should I do if I want to force the redownload of the lyrics? Is there a cache located somewhere that I should delete?

**Comment by Jamy-L** [comment]
My bad, I got it sorted (only after opening the issue of course 🤡 ). It is mandatory to manually start the "Download missing Lyrics" task, making a library refresh is not sufficient. This will create the .lrc if the library options is on, regardless of whether or not a lyrics fetch was already done: there is no need to manualy delete the lyrics.
